ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- set_fact:
+    bootstrap_timeout: "{{ (increase_bootstrap_timeout + 1) }}"
+
+- set_fact:
+    install_timeout: "{{ (increase_install_timeout + 1) }}"
+
```

Comment:
Why is the +1 needed?

Reply:
That is vestigial code from another approach I was considering. I'll clean it up.

Reply:
In the user context of the config file, this is an "increase" value, so it needs to be a timeout in addition to the default, thus incrementing the values by 1 before using them in the below tasks.